test/onnx/torchlib/ops_test_common.py

```diff
@@ -500,10 +508,10 @@ def _capture_graph_and_evaluate_torch_script_evaluator(
             (),
             (),
             nodes=(),
-            opset_imports={"": 18, "pkg.torch.onnx": 1},
+            opset_imports={"": opset_version, "pkg.torch.onnx": 1},
             name="main_graph",
         )
-        opset = onnxscript.opset18
+        opset = OPSET_VERSION_MAPPING[opset_version]
```

Comment:
There is a way to construct the object so we don't need the mapping

```suggestion
        opset = onnxscript.values.Opset(opset_version)
```

I think something like this